Documentation/Signoffs/CANCapturing.md

```diff
@@ -20,9 +20,13 @@ that is captured.
 ### Analysis
 ##### 1. Microcontroller Selection
 The Arduino Leonardo CAN BUS Board is designed for the very purpose of 
-reading CAN bus messages through the OBD port of a vehicle. The board contains the MCP2551 CAN transceiver
-to read the CAN bus at a rate of up to 1 Mbps and is connected through a DB9 to OBDII cable to the Nissan Leaf.
-The arduino leonardo receives power and transmitts data through a standard USB port that connects directly to a PC.
+reading CAN bus messages through the OBD (On-Board Diagnostics) port of a vehicle. The OBDII port is a 16-pin port commonly used to detect issues or malfunctions within a vehicle. However, the OBDII port in the Nissan Leaf also allows access to each of the three CAN bus systems in the vehicle through some of the pins on the port [1].
```

Comment:
In the analysis you need to show that the Arduino has the same pins setup to read the CAN bus pins or show that these connectors both use a common standard so that you are certain this will work.